lib/salus/scanners/npm_audit.rb

```diff
@@ -7,14 +7,14 @@
 
 module Salus::Scanners
   class NPMAudit < NodeAudit
-    AUDIT_COMMAND = 'npm audit --json'.freeze
+    AUDIT_COMMAND = 'pnpm audit --json'.freeze
 
     def should_run?
-      @repository.package_lock_json_present?
+      true
     end
 
     def version
-      shell_return = run_shell('npm audit --version')
+      shell_return = run_shell('pnpm --version')
```

Comment:
This looks like it breaks npm support. Is that desired? I'd expect this to be additive.

Reply:
For the specific purposes of our fork I think this is ok. It would be nice if we could submit a PR to Salus rather than just an issue, but I also understand that we have higher priority things to work on. I'm not super optimistic we'll be able to get off of our fork any time soon though 😢